cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -44,6 +48,22 @@ const Modules = () => {
   useEffect(() => {
     setLoadingModules(true);
 
+    function populateNamespaceData() {
+      axios
+        .get(`/api/namespaces`)
+        .then((res) => {
+          setNamespaceFilterData(res.data);
+          setModuleNamespaceFilter(res.data);
+          setLoadingModules(false);
+        })
+        .catch((error) => {
+          setError(mapResponseError(error));
+          setLoadingModules(false);
```

Comment:
Remove the `setLoadingModules` call from here. It should be set to false only when the modules are loaded.